lib/fastlane/plugin/wpmreleasetoolkit/helper/android/android_localize_helper.rb

```diff
@@ -171,6 +171,22 @@ def self.verify_pr_diff(main, library, main_strings, lib_strings, source_diff)
             end
           end
         end
+
+        # Extract the value of a import key from build.gradle
+        #
+        # @param [String] import_key The key to look for
+        # @return [String] The value of the key, or nil if not found
+        #
+        def self.get_library_version_from_gradle_config(import_key:)
+          gradle_file_path = File.join(ENV['PROJECT_ROOT_FOLDER'] || '.', 'build.gradle')
+
+          return nil unless File.exists?(gradle_file_path)
+
+          File.open(gradle_file_path, 'r') do | f |
+            text = f.read
+            text.match(/#{import_key}\s?=\s?'(.*?)'/m)&.captures&.first
```

Comment:
Alternatively maybe you'd be interested in taking some inspiration on how I do a very similar thing in my WIP branch here: https://github.com/wordpress-mobile/release-toolkit/blob/cleanup/203-version/lib/fastlane/plugin/wpmreleasetoolkit/models/android/version_set.rb#L29-L52 (iow when iterating on each line, call `.strip` on the line being checked to clear leading and trailing whitespaces, then anchor the text to search on the stripped/trimmed line)

Reply:
(though, that being said, I'm also gonna borrow a couple of your ideas in that WIP branch at some point, like the use of `&.captures&.first` instead of my use of `RegExp.last_match` 👍  love that one)